core/src/main/java/com/yahoo/ycsb/workloads/CoreWorkload.java

```diff
@@ -505,7 +525,7 @@ public String buildKeyName(long keynum) {
     if (!orderedinserts) {
       keynum = Utils.hash(keynum);
     }
-    return "user" + keynum;
+	return "user"+String.format("%0"+zeropadding+"d",keynum);
```

Comment:
Done. Testing showed format was ~11x slower compared to original code (no padding) and this new version is only ~20% slower compared to original code, while it does padding.